spec/support/matchers/have_column.rb

```diff
@@ -8,7 +8,13 @@
     column && column.default.to_s == @default.to_s
   end
 
-  failure_message_for_should do |columns|
-    "expected to find '#{column_name}', default '#{@default}' in #{columns.map{|column| [column.name, column.default] }}"
+  if RSpec::Version::STRING.to_i >= 3
+    failure_message do |columns|
+      "expected to find '#{column_name}', default '#{@default}' in #{columns.map{|column| [column.name, column.default] }}"
+    end
+  else
+    failure_message_for_should do |columns|
+      "expected to find '#{column_name}', default '#{@default}' in #{columns.map{|column| [column.name, column.default] }}"
```

Comment:
Line is too long. [123/80]<br>Space missing to the left of {.<br>Space between { and | missing.